Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml

```diff
@@ -114,6 +114,26 @@
   satchel: ClothingBackpackDuffelSyndicateOperative
   duffelbag: ClothingBackpackDuffelSyndicateOperative
 
+#Syndicate Lonoly Operative Outfit - Full Kit
```

Comment:
Why is this a full gear change from the regular operatives? To include the pinpointer?